submissions/bilkisuTace/HTML/product.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>Products | Ladies & Gents</title>
+	<header>	
```

Comment:
<header> tag is meant to be inside the <body> tag. Because it is called header doesn't mean it should be in the <head> tag